portfolio/src/main/webapp/script.js

```diff
@@ -81,58 +81,99 @@ function displayComments() {
     });
 }
 
+function addLandmark(map, lat, lng, title, description) {
+  const marker = new google.maps.Marker(
+      {position: {lat: lat, lng: lng}, map: map, title: title});
+
+  const infoWindow = new google.maps.InfoWindow({content: description});
+  marker.addListener('click', () => {
+    infoWindow.open(map, marker);
+  });
+}
+
+function createInfoWindowContent(name, location, description, url, img) {
+    return '<h2>'+name+'</h2>' + '<h3>'+location+'</h3>' 
+    + '<div class = "clearfix"><img class= "img1" src='+ img +
+    ' width="150" height=auto>' + '<p>'+description+' <a href =\"' +
+    url+ '\" target = "_blank">'+ url+'</a></p></div>' ;
+}
 function createMap() {
   const map = new google.maps.Map(
       document.getElementById('map'),
       {center: {lat: 37.0902, lng: -95.7129}, zoom: 4});
 
-  const gauchoMarker = new google.maps.Marker({
-    position: {lat: 40.4428, lng: -80.0025},
-    map: map,
-    title: 'Gaucho Parrilla Argentina'
-  });
-
-  const pimaanMarker = new google.maps.Marker({
-    position: {lat: 40.9724, lng: -74.0285},
-    map: map,
-    title: 'Pimaan Thai Restaurant'
-  });
-
-  const brendasMarker = new google.maps.Marker({
-    position: {lat: 37.7829, lng: -122.4189},
-    map: map,
-    title: 'Brenda\'s French Soul Food'
-  });
+  addLandmark(
```

Comment:
This seems like a giant pain, but the lines seem outside the 80 column limit in this section.

Reply:
I think you can configure the maximum column limit in Cloud Shell Editor.

Go to File -> Preferences. In the right-half section, which has tabs for `User` and `Workspace` preferences, add the settings entry below:

```
"editor.wordWrapColumn": 80
```